admin/coverage

```diff
@@ -7,4 +7,9 @@ cargo llvm-cov clean --workspace
 
 cargo build --locked --all-targets --all-features
 cargo test --locked --all-features
+
```

Comment:
Bullet lists in a commit message are never a good sign... Maybe split this up?

Reply:
Have split this in two.